packages/plugin-js-packages/src/lib/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { MaterialIcon } from '@code-pushup/models';
+import { PackageManager } from './config';
+
+export const pkgManagerNames: Record<PackageManager, string> = {
+  npm: 'NPM',
+  'yarn-classic': 'Yarn v1',
+  'yarn-modern': 'Yarn v2+',
+  pnpm: 'PNPM',
+};
+
+export const pkgManagerIcons: Record<PackageManager, MaterialIcon> = {
+  npm: 'npm',
+  'yarn-classic': 'yarn',
+  'yarn-modern': 'yarn',
+  pnpm: 'pnpm',
+};
+
+export const pkgManagerDocs: Record<PackageManager, string> = {
+  npm: 'https://docs.npmjs.com/',
+  'yarn-classic': 'https://classic.yarnpkg.com/docs/',
+  'yarn-modern': 'https://yarnpkg.com/getting-started',
+  pnpm: 'https://pnpm.io/pnpm-cli',
+};
+export const auditDocs: Record<PackageManager, string> = {
+  npm: 'https://docs.npmjs.com/cli/commands/npm-audit',
+  'yarn-classic': 'https://classic.yarnpkg.com/docs/cli/audit',
+  'yarn-modern': 'https://yarnpkg.com/cli/npm/audit',
+  pnpm: 'https://pnpm.io/',
+};
+
+export const outdatedDocs: Record<PackageManager, string> = {
+  npm: 'https://docs.npmjs.com/cli/commands/npm-outdated',
+  'yarn-classic': 'https://classic.yarnpkg.com/docs/cli/outdated/',
+  'yarn-modern': 'https://github.com/mskelton/yarn-plugin-outdated',
+  pnpm: 'https://pnpm.io/cli/outdated',
+};
+
+export const groupDepDocs: Record<PackageManager, string> = {
+  npm: 'https://docs.npmjs.com/cli/using-npm/config#include',
+  'yarn-classic': 'https://classic.yarnpkg.com/docs/usage',
+  'yarn-modern': 'https://yarnpkg.com/cli/npm/audit#options',
+  pnpm: 'https://pnpm.io/cli/audit#options',
+};
```

Comment:
These are best official docs on dependency types I could find: https://classic.yarnpkg.com/en/docs/dependency-types

The concept is the same for all packagers. So instead having different links per package manager, I would create different links per dependency type:
- `prod`: https://classic.yarnpkg.com/en/docs/dependency-types#toc-dependencies
- `dev`: https://classic.yarnpkg.com/en/docs/dependency-types#toc-devdependencies
- `optional`: https://classic.yarnpkg.com/en/docs/dependency-types#toc-optionaldependencies

Reply:
good point